e2e/tests/rpaHandoff_test.js

```diff
@@ -60,8 +60,8 @@ Scenario('Claimant does not respond to defence with defined timescale', async (I
   await waitForFinishedBusinessProcess(caseId);
   await updateCaseData(caseId, {applicantSolicitorSecondResponseDeadlineToRespondentSolicitor1: dateTime(-1)});
```

Comment:
case field should be updated to `claimDismissedDeadline`

Reply:
oops good spot, new good since I raised the PR I think!